rcar_flash/rcar_flash.py

```
@@ -221,12 +222,23 @@ def do_flash(conf, args):  # noqa: C901
     if args.flash_writer:
         if not args.cpld:
             log.info("Please ensure that board is in the serial download mode")
+
         if args.flash_writer == "DEFAULT":
-            fname = board["flash_writer"]
+            resource_name = board["flash_writer"]
         else:
-            fname = args.flash_writer
-        if not os.path.exists(fname):
-            raise Exception(f"Flash write file {fname} does not exists!")
+            resource_name = args.flash_writer
+
+        try:
+            resource_path = files("rcar_flash").joinpath(resource_name)
+        except Exception:
+            raise Exception(f"Flash writer file {resource_name} does not exist in package resources.")
```

Do you really need to hide original exception here?